tck/README.md

```diff
@@ -479,6 +479,22 @@ public class SkippingIdentityProcessorTest extends IdentityProcessorVerification
 }
 ```
 
+## Which verifications must be implemented by an compliant implementation?
+In order to be considered an Reactive Streams compliant we require implementations to cover their
```

Comment:
I'm trying to answer the question @purplefox had here, paraphrasing it: "which tests do I need to implement for you guys to believe me my framework works with reactive streams?"

Do you think the "compliant" word is confusing or a good one here?